### PR TITLE
Adding the release notes for v1.3

### PR DESCRIPTION
**Pull request by aburdenthehand**
Prepared to be released with KubeVirt v1.3 on July 3rd

@xpivarc @mhenriks @EdDev @lyarwood - if you could please peruse the relevant sections to ensure accuracy

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -5,6 +5,125 @@ hide:
...
+- [[PR #11256]](https://github.com/kubevirt/kubevirt/issues/11256) [andreabolognani] This version of KubeVirt includes upgraded virtualization technology based on libvirt 10.0.0 and QEMU 8.2.0.
+- [[PR #11788]](https://github.com/kubevirt/kubevirt/issues/11788) [ormergi] The network-info annotation is now used for mapping between SR-IOV network and the underlying device PCI address
+- [[PR #11659]](https://github.com/kubevirt/kubevirt/issues/11659) [iholder101] Require scheduling infra components onto control-plane nodes
+- [[PR #12079]](https://github.com/kubevirt/kubevirt/issues/12079) [EdDev] Network hotplug feature is declared as Beta.
+
+### SIG-infra
+- [[PR #11648]](https://github.com/kubevirt/kubevirt/issues/11648) [kubevirt-bot] Updated common-instancetypes bundles to v1.0.0
+- [[PR #11962]](https://github.com/kubevirt/kubevirt/issues/11962) [lyarwood] `VirtualMachines` referencing an instance type are now allowed when the `LiveUpdate` feature is enabled and will trigger the `RestartRequired` condition if the reference within the `VirtualMachine` is changed.
```
Comment (lyarwood) [inline docs/release_notes.md:88]: We can drop this as it was immediately replaced with the actual implementation on the line below, this was just a bugfix we backported to release-1.2 first.

File: docs/release_notes.md [hunk 2]
```diff
@@ -5,6 +5,125 @@ hide:
...
+- [[PR #11455]](https://github.com/kubevirt/kubevirt/issues/11455) [lyarwood] `LiveUpdates`  of VMs using instance types are now supported with the same caveats as when making changes to a vanilla VM.
+- [[PR #11681]](https://github.com/kubevirt/kubevirt/issues/11681) [lyarwood] The `CommonInstancetypesDeployment` feature and gate are retrospectively moved to Beta from the 1.2.0 release.
+- [[PR #12171]](https://github.com/kubevirt/kubevirt/issues/12171) [kubevirt-bot] `PreferredDiskDedicatedIoThread` is now only applied to `virtio` disk devices
```
Comment (lyarwood) [inline docs/release_notes.md:91]: This was my bugfix backported after the branch was cut but it's not a massive issue.

File: docs/release_notes.md [hunk 3]
```diff
@@ -5,6 +5,125 @@ hide:
...
+### SIG-network
+- [[PR #11653]](https://github.com/kubevirt/kubevirt/issues/11653) [EdDev] Build the `passt`custom CNI binary statically, for the `passt` network binding plugin.
+- [[PR #11678]](https://github.com/kubevirt/kubevirt/issues/11678) [Vicente-Cheng] Improve the handling of ordinal pod interface name for upgrade
+- [[PR #11618]](https://github.com/kubevirt/kubevirt/issues/11618) [AlonaKaplan] Extend network binding plugin to support device-info DownwardAPI.
+- [[PR #11256]](https://github.com/kubevirt/kubevirt/issues/11256) [andreabolognani] This version of KubeVirt includes upgraded virtualization technology based on libvirt 10.0.0 and QEMU 8.2.0.
+- [[PR #11788]](https://github.com/kubevirt/kubevirt/issues/11788) [ormergi] The network-info annotation is now used for mapping between SR-IOV network and the underlying device PCI address
+- [[PR #11659]](https://github.com/kubevirt/kubevirt/issues/11659) [iholder101] Require scheduling infra components onto control-plane nodes
+- [[PR #12079]](https://github.com/kubevirt/kubevirt/issues/12079) [EdDev] Network hotplug feature is declared as Beta.
+
+### SIG-infra
```
Comment (lyarwood) [inline docs/release_notes.md:86]: There's no such SIG as `SIG-infra` FWIW, technically the following should all be under `SIG-compute`.

**Comment by aburdenthehand** [comment]
Thanks @lyarwood. I've moved your bugfix and folded the infra notes into sig-compute.

**Comment by aburdenthehand** [comment]
Additional bugs for rc2 have been added

**Comment by aburdenthehand** [comment]
@jean-edouard @stu-gott FYI v1.3.0 has now been tagged

File: docs/release_notes.md [hunk 4]
```diff
@@ -5,6 +5,128 @@ hide:
 
 # KubeVirt release notes
 
+## v1.3.0
+
+Release on: Wed Jul 03 2024
```
Comment (jean-edouard) [inline docs/release_notes.md:10]: Can you please just fix the date?

Comment (aburdenthehand) [reply docs/release_notes.md]: Done and done - thanks for the catch

**Review by jean-edouard — APPROVED** [review]

**Pull request merged**